staging/src/k8s.io/code-generator/pkg/util/build.go

```diff
@@ -50,3 +51,24 @@ func hasSubdir(root, dir string) (rel string, ok bool) {
 	// cut off root
 	return filepath.ToSlash(dir[len(root):]), true
 }
+
+// GetBoilerplatePath uses the boilerplate in k8s.io/code-generator.
+// If it does not exist, it uses the boilerplate in k8s.io/kubernetes.
+func GetBoilerplatePath() string {
+	boilerPlatePath := "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt"
```

Comment:
Wondering, do we really want this path here? Shouldn't we add it explicitly in all calls in kube? How many are that?